Docs: request security (#30937)
grafana/grafana: docs/sources/enterprise/enterprise-configuration.md

### lease_renewal_increment

New duration for renewed tokens. Vault may be configured to ignore this value and impose a stricter limit.

## [security.egress]

> > **Note:** Available in Grafana Enterprise v7.4+.

Reviewer: Could you add a brief description of what these settings do? I'm guessing from the name something about outgoing security?

### host_deny_list

> A list of hostnames or IP addresses separated by spaces for which requests will be blocked.

Reviewer 2: ```suggestion
A list of hostnames or IP addresses separated by spaces for which requests are blocked.
```

Reviewer 2: The default in tech writing is to write in the present tense for general things that now and in general. Unless there is a specific event that must happen in the future, use present tense.

Author: Right, thanks.

### host_allow_list

> A list of hostnames or IP addresses separated by spaces for which requests will be allowed, all other requests will be blocked.

Reviewer 2: ```suggestion
A list of hostnames or IP addresses separated by spaces for which requests are allowed. All other requests are blocked.
```

### header_drop_list

> A list of headers that will be stripped from outgoing datasource and alerting requests.

Reviewer 2: ```suggestion
A list of headers that are stripped from the outgoing data source and any alerting requests.
```

Author: Not sure I understand the `any` here. What are we going for and is there a case for making it `A list of headers that are stripped from the any outgoing data source and alerting requests.`?

Reviewer 2: Got it. Reverting my change. I appreciate the pushback.

Reviewer 2: Suggestion edited for your convenience.

Author: Thanks!

> ### cookie_drop_list =

Reviewer: Is the equal sign supposed to be here? I haven't seen that on any of the other setting headings.

Author: No, thanks.

> A list of cookies that will be stripped from outgoing datasource and alerting requests.

Reviewer 2: ```suggestion
A list of cookies that are stripped from the outgoing data source and any alerting requests.
```
